### ISSUE TITLE
Copying cache from tmp

### ISSUE THREAD

**Issue by mbopp** [symfonycorp/platformsh-symfony-template#8]
In `symfony-deploy` > `symfony-start`, the newly built cache, which is stored in the the `tmp` directory is then copied to it's correct location inside `/var/cache...`.

I am noticing in my deploy logs that I get many errors like the following....

```
cp: cannot create directory 'var/cache/prod/pools/system/cl7zdSPa-x/M/W': File exists
cp: cannot create directory 'var/cache/prod/pools/system/cl7zdSPa-x/Z/Y': File exists
cp: cannot create directory 'var/cache/prod/pools/system/kIX5KW0KHn': File exists
cp: cannot create directory 'var/cache/prod/pools/system/mjux7ahRje/G': File exists
cp: cannot create directory 'var/cache/prod/pools/system/mjux7ahRje/J': File exists
cp: cannot create directory 'var/cache/prod/pools/system/mjux7ahRje/N': File exists
cp: cannot create regular file 'var/cache/prod/url_generating_routes.php': File exists
cp: cannot create regular file 'var/cache/prod/url_matching_routes.php': File exists
```

Are these benign, or is it something that indicates a problem that needs to be dealt with?

I also see that at the beginning of `symfony-start` the cache directory is cleared via `rm -rf var/cache/* app/cache/*`
So I'm not sure why this problem exists.

I do, sometimes see messages like ... 

```
rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV': Directory not empty
```

- or -

```
    W: + symfony-start

    W: + rm -rf var/cache/prod 'app/cache/*'

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/-/Y/DBsTv75ZjZ7YauIClaaw': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/-/6/3Rl-IW8jd20pylX1f9CA': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/4/S/99CCsiAT3PjCuUnQ5EXA': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/4/A/0hx0j2JVHjWNm261wBKg': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/4/+/gt1OYW6-3h7c7oUnQGDw': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/4/F/42abNtBO25Ns8W20T1nw': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/4/M/2c6rYsqYjS2MJshOTxNg': Stale file handle

    W: rm: cannot remove 'var/cache/prod/pools/system/+F8blG3ckV/A/-/2yseyuaH1zWYiWQiNmjw': Stale file handle
```

Any help or clarity would be greatly appreciated.

**Comment by mbopp** [comment]
It might be premature, but this might be related to have the `/var` directory mounted as a `network-storage` service. I switched it to a local mount and have not yet encountered these problems. I don't quite know why this would be, as I imagined Network Storage mounts to work most the same as a local one.  Luckily, I think I no longer have the need for this to be on the Network Storage, so hopefully this makes a difference as far as the cache stuff goes.
